android/app/src/main/AndroidManifest.xml

```diff
@@ -123,6 +123,17 @@
         <data android:host="omaps.app"/>
       </intent-filter>
 
+      <intent-filter android:autoVerify="@bool/autoVerify">
```

Comment:
What does autoVerify mean? "openstreetmap.org" doesn't have it.

Reply:
As we can't load a custom json file to osm.org website, autoVerify won't work https://developer.android.com/training/app-links/verify-android-applinks

So this attribute should be removed.